File: biometric/src/main/java/dev/skomlach/biometric/compat/impl/AuthCallback.java
package dev.skomlach.biometric.compat.impl;

import androidx.annotation.RestrictTo;

@RestrictTo(RestrictTo.Scope.LIBRARY)
public interface AuthCallback {
    void startAuth();

    void stopAuth();

    void cancelAuth();

    void onUiOpened();

    void onUiClosed();
}

File: biometric/src/main/java/dev/skomlach/biometric/compat/impl/AuthCallback.kt
package dev.skomlach.biometric.compat.impl

import androidx.annotation.RestrictTo

@RestrictTo(RestrictTo.Scope.LIBRARY)
interface AuthCallback {
    fun startAuth()
    fun stopAuth()
    fun cancelAuth()
    fun onUiOpened()
    fun onUiClosed()
}